app/classes/api/observation_api.rb

```diff
@@ -6,27 +6,23 @@ class API
   class ObservationAPI < ModelAPI
     self.model = Observation
 
-    self.high_detail_page_length = 10
-    self.low_detail_page_length  = 100
+    self.high_detail_page_length = 100
+    self.low_detail_page_length  = 1000
     self.put_page_length         = 1000
     self.delete_page_length      = 1000
 
     self.high_detail_includes = [
-      :comments,
-      { images: :license },
-      :location,
-      :name,
-      { namings: [:name, { votes: :user }] },
       :collection_numbers,
+      { comments: :user },
```

Comment:
Does this mean that only the users who commented are returned or that both the comment and now the user who created that comment is included?

Reply:
No, I must've confused the diff, because it still includes comments and the users who made the comments like always.